db/migrations/000001_init.up.sql

```diff
@@ -181,6 +181,34 @@ CREATE TABLE IF NOT EXISTS user_answers (
         REFERENCES answer_alternatives(question_id, id)
 );
 
+-- calculates points awarded for a question based on the time spent on the question, question max points and duration
```

Comment:
Comment says "time spent on the question" and "duration"; what are the differences?

Reply:
by duration i mean the question duration, refers to `time_limit_seconds` on the questions table. can clarify